src/universal/modules/meeting/components/MeetingLobby/MeetingLobby.js

```diff
@@ -25,7 +25,7 @@ const MeetingLobby = (props) => {
   const {members, team, styles} = props;
   const {id: teamId, name: teamName} = team;
   const onStartMeetingClick = createStartMeetingHandler(members);
-  const shortUrl = makeHref(`/team/${teamId}`);
+  const meetingUrl = makeHref(`/meeting/${teamId}`);
```

Comment:
No concern for the `const` naming. But just FYI, we do have a short URL `prbl.io` and we have a backlog issue to use here #827 #future

Reply:
Ahh, I see. It's a total OCD thing, but I liked getting `meeting` in there, because the URL does semantically refer to the meeting. Perhaps when #827 lands, we can call it something like `shortMeetingUrl` 🙂